modules/ROOT/pages/_partials/flex-tls-cipher.adoc

```diff
@@ -0,0 +1,44 @@
+//tag::intro1[]
+== TLS Support on Flex Gateway
+
+Flex can support a range of TLS Ciphers from TLS 1.1 to TLS 1.3. You cannot support both TLS 1.1 and TLS 1.3 without also supporting TLS 1.2. The default minimum and maximum values for Flex Gateway connected mode are both 1.3. 
+
+You cannot customize the list of TLS 1.3 Ciphers.
```

Comment:
We need to specify somewhere the actual list of 1.3 ciphers that each version of flex supports. That could be here in this doc or somewhere else (release notes maybe?) that we can link to from this page.